### DISCUSSION TITLE
Resources from MVP session 2: Building a Chat on your Business Data without writing a line of code

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/AI-Chat-App-Hack#88] [Announcements]
Links:
- https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-4-and-gpt-4-turbo-preview-model-availability
- https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models
- https://learn.microsoft.com/en-us/azure/ai-services/openai/use-your-data-quickstart?tabs=command-line%2Cpython&pivots=programming-language-studio
- https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant

Demo Repo: https://github.com/Azure-Samples/azure-search-openai-demo/

Video: https://www.youtube.com/watch?v=6p5CBzS_qQI

**Comment by zzulueta** [comment]
What would be the criteria to determine if you should use code to develop the chat app vs. using the studio?

**Reply by zedhaque** [reply]
If you want to make 
- extensive UI customisation 
- modify the prompt as per your liking 
- modify methods or functions before or after the chat completion with the user
- custom indexing of your data
- In short customisation 
IMHO studio is directed to enterprise customers who just want to chat with their own data. I am not sure if studio lets you launch your site publicly (outside enterprise firewall).
Note, You can download the source code of your studio instance and modify it later. So it’s pretty good for MVP without coding.

**Comment by zzulueta** [comment]
Why did we demo Azure OpenAI Studio and not the newer Azure AI Studio? What would be the difference of the two?

**Reply by pamelafox** [reply]
1) The newer Azure AI Studio is very much in flux these days, so it can be hard to keep up with the UI and know where to go. So old-timers like me might be tempted to stick with Azure OpenAI Studio.
2) I believe the Azure AI Studio requires the creation of an "Azure AI Hub" and and "Azure AI Project" in order to use the tools inside, and those are not required for Azure OpenAI Studio. You only need an Azure OpenAI resource to use Azure OpenAI Studio.
